### PR TITLE
Adding hints to parse for students.

### PR DESCRIPTION
**Pull request by rawrgulmuffins**
### Description

Based on student feedback I've added some hints to parse to make things less difficult.

### CODE REVIEW

File: calc/calc.py [hunk 1]
```diff
@@ -120,7 +120,15 @@ def parse(self):
...
                 of arithmetic operations.
         """
-        # This method should make use of ``_consume_token``. For example usage
-        # of that method please see the ``test_calc_can_consume_valid_token``
-        # test in ``tests/test_calc.py`` on line #53.
-        pass
+        # Hint Assignment Two:
+        #    This method will have to call `_consume_token` three times to get
+        #    1+1 to work correctly.
+        left_token = consume(INTEGER)
```
Comment (reillysiemens) [inline calc/calc.py]: Is `consume` just a function now?

File: calc/calc.py [hunk 2]
```diff
@@ -120,7 +120,15 @@ def parse(self):
...
+
+        return_value = None
+
+        # Rabbit Hole:
```
Comment (reillysiemens) [inline calc/calc.py]: Good `Rabbit Hole`.

File: calc/calc.py (hunk 1 above)
Comment (rawrgulmuffins) [inline calc/calc.py]: Excuse me while I'm hyper and bad.

**Comment by reillysiemens** [comment]
LGTM :ship:

**Pull request merged**